### ISSUE TITLE
Create open text section for alt text at the image level in order manager.

### ISSUE THREAD

**Issue by kelea99** [pulibrary/figgy#6729]
### Summary (Product Owner)
Alt text has been discussed as a need by ephemera stakeholders, originally thought to be a potential field addition in the ephemera form. however, conversations with @tpendragon and influence from spotlights additions to widgets in our instance of DPUL have raised the idea of adding alt text at the image level as part of order manager options. It also should be one line/simple text. I wonder if a character limit should also be put in place. A best practice for writing alt text might also be cool here. Perhaps talk with Jane and Christina?
### User Story (Product Owner, when applicable)
_As a [persona], I want to [describe intent], so that [explain the big problem you are trying to solve and its benefit]_
As a figgy user, I would like the opportunity to add alt text to images so that I may broaden access to our collections in rich and inclusive ways.  
As a DPUL-C software developer, I want to be able to index the alt text field so that I can refine and broaden connections in our "related items" section.

### Impact (Product Owner)
I believe the impact is two fold. externally,  we are opening our digital collections to more people. we are democratizing access. If our goal is to reach classrooms, it should be in service to all students sitting in that room. descriptors of the images would enrich the experience for all.  I also believe this will have an impact on our applications. If we index the descriptors, it may have a direct (and positive) impact on related items our visitors and researchers may find of interest, that they would not even know we had. 

### Priority recommendation (Product Owner)

- [ ] asap
- [ ] within the next 3 weeks
- [ ] PO will prioritize

### Implementation Notes
_To be filled out by DLS._

### Acceptance Criteria
_To be filled out by DLS.Some requirements to consider, including:_

- [ ] The UI implemented for this issue meets accessibility requirements
- [ ] New functionality is documented

### First Step
_To be filled out by DLS._

### Screenshots (Product Owner)
Example from DPUL:

<img width="1512" alt="Image" src="https://github.com/user-attachments/assets/64bf4631-074e-4376-80ba-588592228137" />
